README.md

```diff
@@ -0,0 +1,30 @@
+## typora
```

Comment:
In general, we try to avoid adding steps which basically duplicate existing documentation.